backend-server/src/api/controllers/availabilityController.ts

```diff
@@ -15,6 +15,10 @@ export interface GetAvailabilityBody {
   startTime: string;
 }
 
+export interface getAllAvailabilitiesBody {
```

Comment:
Also we should try to capitalize class/interface names, and only use lowercase for function names, but we can change this in a separate PR